spooky-oysters/Bangazon-Command-Line-Application
# Dr active customer

## Link to Ticket
[Make a Customer Active](https://github.com/spooky-oysters/Bangazon-Command-Line-Application/issues/2)

## Description of Proposed Changes

- Bangazonians should be able to assign an active customer from the main menu

## Steps to Test

```sh
git fetch --all
git checkout dr-ActiveCustomer
cd bangazon-cli.Tests/
dotnet test
```
- Assuming the tests pass, navigate to `bangazon-cli.Tests/ActiveCustomer_Should.cs`
- Check that the functionality for the tests and the methods associated follow the guidelines of the ticket 

## Impacted Areas in Application

* Customer profiles and associated customer data including products, orders, payments, etc.
* Managers/ActiveCustomerManager.cs 

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each.
